src/Features/LanguageServer/Protocol/Handler/Hover/HoverHandler.cs

```diff
@@ -47,17 +51,42 @@ public HoverHandler()
                 return null;
             }
 
-            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            var supportsVSExtensions = context.ClientCapabilities.HasVisualStudioLspCapability() == true;
 
-            // TODO - Switch to markup content once it supports classifications.
-            // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/918138
-            return new VSHover
+            var hover = await GetHoverAsync(info, document, supportsVSExtensions, cancellationToken).ConfigureAwait(false);
```

Comment:
Usually I feel weird about having an intermediary method like this which isn't re-used; however, I also like the fact that you can inspect the hover at debug times easily. Conflicted! 😄

Reply:
yeah this was intentional to allow me to debug it.